prov/efa/src/rxr/rxr_cq.c

```diff
@@ -579,6 +579,10 @@ void rxr_cq_proc_pending_items_in_recvwin(struct rxr_ep *ep,
 
 	while (1) {
 		pending_pkt = *ofi_recvwin_peek((&peer->robuf));
+		/* Coverity will complain that comparing an array to null is not
+		 * useful, but we use this structure as the header of a compound
+		 * object, so it is correct and useful.
+		 */
 		if (!pending_pkt || !pending_pkt->pkt)
```

Comment:
I do not think the comment is correct here. The checking for `!pending_pkt` is valid.

The complain of coverity is about `!pending_pkt->pkt`, which is valid because `pending_pkt->pkt = (char *)pending_pkt + 64`, which is always > 0. So the correct course is to remove `!pending_pkt->pkt`.

Reply:
Thanks, I'll change it